proposals/dao/fip_41.ts

```diff
@@ -0,0 +1,156 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '../../types/types';
+import { TransactionResponse } from '@ethersproject/providers';
+import { expectApprox, getImpersonatedSigner } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+chai.use(CBN(ethers.BigNumber));
+
+// Constants
+// LBP swapper
+const MIN_LBP_SIZE = ethers.constants.WeiPerEther.mul(50_000); // 50k FEI
+
+/*
+
+TRIBE Buybacks
+
+DEPLOY ACTIONS:
+
+1. Deploy LUSD Fuse deposit
+2. Deploy LUSD LBP Swapper
+3. Create LUSD LBP pool
+4. Init LUSD LBP Swapper
+
+DAO ACTIONS:
+1. Mint 1.1M FEI to Timelock
+2. Approve 1.1M FEI to trade on Saddle's D4 pool
+3. Swap 1.1M FEI for LUSD on Saddle's D4 pool
+4. Transfer LUSD to the LBP Swapper
+5. Mint 100M FEI for LBP Swapper
+6. Start auction
+*/
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  const { core, fei, lusd, oneConstantOracle, balancerLBPoolFactory } = addresses;
+
+  if (!core) {
+    console.log(`core: ${core}`);
+
+    throw new Error('An environment variable contract address is not set');
```

Comment:
nit: this isn't an environment variable.

Reply:
prob leave as is